app/src/main/java/com/babylon/wallet/android/presentation/transaction/TransactionReviewViewModel.kt

```diff
@@ -285,12 +287,25 @@ class TransactionReviewViewModel @Inject constructor(
         val error: TransactionErrorMessage? = null,
         val ephemeralNotaryPrivateKey: Curve25519SecretKey = Curve25519SecretKey.secureRandom(),
         val selectedFeePayerInput: SelectFeePayerInput? = null,
-        val hiddenResourceIds: PersistentList<ResourceIdentifier> = persistentListOf()
+        val hiddenResourceIds: PersistentList<ResourceIdentifier> = persistentListOf(),
+        val transactionManifestData: TransactionManifestData? = null
```

Comment:
Why do we now need to keep the manifest data in state? Just a question. I don't remember why we didn't need to keep it in the past? I think we could retrieve it based on the `request`? Do we actually need it in the state?

Reply:
I think it's just for convenience, to keep somehwere the mutated transaction manifest. But, I can see that we only mutate it before submitting: first by attaching guarantees and then adding the lock fee if needed, so there is no need to keep the resulting manifest in the state. I'll double-check and eventually make this change as part of the refactoring [PR](https://github.com/radixdlt/babylon-wallet-android/pull/1223).

Reply:
Yes, exactly that was the case.